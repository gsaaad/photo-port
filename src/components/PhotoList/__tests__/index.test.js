/* eslint-disable testing-library/prefer-screen-queries */
import React from "react";
import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";

import PhotoList from "..";

const example = {
  name: "Grocery aisle",
  category: "commercial",
  description:
    "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc ultricie",
};

afterEach(cleanup);

describe("Render PhotoList component", () => {
  // does photolist render
  it("renders", () => {
    render(<PhotoList example={example} />);
  });
  //   does it match snapshot
  it("matches snapshot", () => {
    const { asFragment } = render(<PhotoList example={example} />);
    expect(asFragment()).toMatchSnapshot();
  });
  // does it render example properties accordingly
  //   it("renders example properties", () => {
  //     const { getByTestId } = render(<PhotoList example={example} />);
  //     expect(getByTestId("h1Title")).toHaveTextContent("Grocery aisle");
  //   });
});
