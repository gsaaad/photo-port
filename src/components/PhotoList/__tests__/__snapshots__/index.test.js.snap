// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render PhotoList component matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="flex-row"
    />
  </div>
</DocumentFragment>
`;
